src/main/java/com/rb/nonbiz/math/stats/NormalDistribution.java

```diff
@@ -61,8 +61,8 @@ public void sanityCheckContents() {
       RBPreconditions.checkNotNull(standardDeviation);
 
       RBPreconditions.checkArgument(
-          standardDeviation > 1e-8,
-          "Standard deviation cannot be almost 0 or negative; was %s",
+          standardDeviation >= 0,
```

Comment:
we have to allow 0 to model situations where the 'distribution' is really just a fixed value.